bedrock/firefox/templates/firefox/nothing-personal/index.html

```diff
@@ -0,0 +1,206 @@
+                  }
+                },
+                {
+                  'media': '(prefers-reduced-motion: no-preference)',
+                  'srcset': {
+                    'img/firefox/nothing-personal/thug-life.gif': '200w'
+                  },
+                  'sizes': {
+                    'default': '200px'
+                  }
+                }
+              ],
+              optional_attributes={
+                  'height': '200',
+                  'loading': 'lazy',
+                  'width': '200'
+              }
+            ) }}
+          </div>
+
+         {% call browser_border(class='mzp-l-content mzp-t-content-md mzp-t-content-nospace', heading='SFAQs', aria_label='Somewhat Frequently Asked Questions') %}
+          <div class="c-browser-window-top">
+            <h4><span aria-hidden="true">[</span>Somewhat Frequently Asked Questions<span aria-hidden="true">]</span></h4>
+
+            <h5>Are you even any good?</h5>
+            <p>Calm down. And yes. <a href="{{ url('firefox.features.fast') }}">We’re so fast</a> you won’t even notice the difference.</p>
+
+            <h5 aria-label="What makes you so special?">What makes you soooo special?</h5>
+            <p><abbr title="No big deal">NBD</abbr>, but people dump their other browsers <a href="{{ url('firefox.features.index') }}">for our features</a> (like <a href="{{ url('firefox.features.picture-in-picture') }}">popping out videos</a> and a <a href="{{ url('firefox.features.pdf-editor') }}">built-in PDF editor</a>).</p>
+
+            <h5>What does any of this have to do <br> with me?</h5>
+            <p>Hey, you can customize Firefox to be all about you with <a href="{{ url('firefox.features.add-ons') }}">add-ons</a> and <a href="{{ url('firefox.features.customize') }}">themes</a>. Happy?</p>
+
+            <h5>Ok, what’s the privacy-catch?</h5>
+            <p><a href="{{ url('firefox.privacy.index') }}">We don’t keep track of where you go</a> because 1. Yawn, and 2. We really don’t need it.</p>
+          </div>
+          <div class="c-sticky-note c-attached-sticky">
+
+           <span aria-hidden="true">>>></span> {{ download_firefox_thanks(alt_copy='download', button_class='mzp-t-primary mzp-t-lg c-download-firefox-cta', download_location='secondary cta') }} <span aria-hidden="true"> <<<< </span>
+           <br>
+           <span aria-hidden="true">>>></span> {{ download_firefox_thanks(alt_copy='download', button_class='mzp-t-primary mzp-t-lg c-download-firefox-cta', download_location='secondary cta') }} <span aria-hidden="true"> <<<< </span>
+           <br>
+           <span aria-hidden="true">>>></span> {{ download_firefox_thanks(alt_copy='download', button_class='mzp-t-primary mzp-t-lg c-download-firefox-cta', download_location='secondary cta') }} <span aria-hidden="true"> <<<< </span>
+            <p>Ok, bye.</p>
+
+          </div>
+        </div>
+        {% endcall %}
+    </div>
+
+    <div class="c-sticky-note c-detached-sticky">
```

Comment:
**issue:** when the page loads on mobile layout, the detached sticky note animates in from the right, over to the left side, and then jumps back to the middle:

![image](https://github.com/user-attachments/assets/307a983e-ad1e-4ccd-84e0-8f8b25553d62)